test/fuzz/random.h

```diff
@@ -33,10 +33,19 @@ class PsuedoRandomGenerator64 : public RandomGenerator {
 } // namespace Random
 
 namespace Fuzz {
+
 class ProperSubsetSelector {
 public:
   ProperSubsetSelector(const std::string& random_bytestring)
-      : random_bytestring_(random_bytestring) {}
+      : random_bytestring_(random_bytestring) {
+    // Must be at least 4 bytes wide to serve as randomness for the subset selector
+    ASSERT(random_bytestring_.length() >= 4);
+    // Pull off the last few bits to make random_bytestring_ a multiple of 4 - this will make
+    // iteration through the bytestring a lot easier and cleaner
+    while (random_bytestring_.length() % 4 != 0) {
+      random_bytestring_.pop_back();
```

Comment:
Could you just make the underlying type a `repeated uint32` to solve this structurally?

Reply:
Changed! Great idea!